core/net/rpl/rpl-icmp6.c

```diff
@@ -1021,12 +1543,20 @@ dao_input(void)
   }
 
   if(RPL_IS_STORING(instance)) {
-    dao_input_storing();
```

Comment:
Stupid question, if RPL_SECURITY is defined (compiled in), is there a mode where I don't set a key and operate insecurely?  Can I do it differently on different interfaces?